src/note/tests/test_note_api.py

```diff
@@ -86,7 +86,11 @@ def test_cannot_create_shared_note_manually(self):
         # NOTE: Should only be able to created SHARED note by inviting useres
         self.assertNotEqual(created_note['access'], 'SHARED')
 
-    def test_note_editor_cannot_invite_others(self):
+    def test_note_editor_can_invite_others(self):
```

Comment:
@yattias is a note editor supposed to be able to invite others? The old test name suggests they shouldn't be able to.

Reply:
I am not familiar with the business rules around this. I renamed the test to reflect the behavior of the current implementation. It allows for this via the `IsOrganizationUser` permission class on the `invite_user` function.

Reply:
Sorry, this was out of scope of this PR. I was just wondering if we should file a bug to fix later.